RESTResource/RESTResource.js

```diff
@@ -662,6 +675,42 @@ export default class RESTResource {
     };
   }
 
+  // Fetches a single page by offset adding it to the existing result list in redux
+  fetchPageByOffset = (options, total) => {
+    const { headers, records, resultOffset, offsetParam } = options;
+    const reqd = Math.min(resultOffset, total);
+    return (dispatch) => {
+      const newOptions = {};
+      newOptions.params = {};
+      newOptions.params[offsetParam] = reqd;
+      const url = urlFromOptions(_.merge({}, options, newOptions));
+      fetch(url, { headers })
+        .then((response) => {
+          if (response.status >= 400) {
+            dispatch(this.fetchHTTPError(response));
+          } else {
+            response.json().then((json) => {
+              const meta = {
+                url: response.url,
+                headers: response.headers,
+                httpStatus: response.status,
+                other: records ? _.omit(json, records) : {},
+              };
+              if (meta.other) meta.other.totalRecords = extractTotal(json);
+              const data = (records ? json[records] : json);
+              dispatch(this.actions.accFetchSuccess(meta, data));
```

Comment:
Great suggestion! I had considered using object keys to store noncontiguous results but felt that it would break the contract modules already have with stripes-connect. A sparse array would allow for the contract to remain with only MCL needing to be aware of the difference.